src/test/javascript/spec/helpers/mocks/service/mock-exercise-hint.service.ts

```diff
@@ -11,11 +11,11 @@ export class MockExerciseHintService implements IExerciseHintService {
         return of({ body: this.exerciseHintDummy }) as Observable<ExerciseHintResponse>;
     }
 
-    delete(exerciseHintId: number): Observable<HttpResponse<void>> {
+    delete(exerciseId: number, exerciseHintId: number): Observable<HttpResponse<any>> {
```

Comment:
Please avoid `any`. 
Why this change anyway? Shouldn't a deletion return a void response?